docs/recipes/add_model_setup.rst

```diff
@@ -310,9 +348,83 @@ Implement a New Coupled Setup
     $ cd ~/model_codes
     $ esm_master install-your_setup-version
 
-8. If everything works correctly you can check that your changes pass `flake8`, commit your changes, push
-   them to the ``origin`` and submit a pull request through GitHub (see steps 5-7 in
-   :ref:`contributing:Contribution to esm_tools Package`).
+10. If everything works correctly you can check that your changes pass ``flake8``, commit your changes, push
+    them to the ``origin`` and submit a pull request through GitHub (see steps 5-7 in
```

Comment:
Line 351/352: I would add here something like "push them to the `origin` remote repository and ..."